CRAWLR/API/ApiHelper.swift

```diff
@@ -47,4 +47,20 @@ class ApiHelper {
             }
         }
     }
+    
+    func getBusiness(id: String, callback: @escaping (Business?) -> Void) {
+        let url = URL(string: ApiHelper.yelpBusinessesUrl + "/\(id)")!
```

Comment:
You don't need to specify the id in the URL string, that's what the parameters object is for.